sharding/contracts/README.md

```diff
@@ -5,6 +5,6 @@ Generate contract bindings from the sharding package with go generate:
 ```bash
 
 cd ..
-go generate
```

Comment:
Why did you change this? `go generate` is the preferred method of generating these bindings. Search across the project for "go:generate" comments